### ISSUE TITLE
Safest Route (as opposed to Shortest Distance and Highest Speed)

### ISSUE THREAD

**Issue by judopp** [sketch-city/project-ideas#83]
Recently, I had an accident on I-45 caused by a pipe-fitting falling off of a truck in front of me.  Traffic was heavy and I made a snap decision to drive over the part rather than swerve and hit another vehicle.  In the process, I took out my suspension (expensive repair but no one was hurt).  Apparently, this is not an isolated incident according to a Houston Chronicle article.  

It occurred to me that if there was a record of such incidents (and other accident data), one could score a Route Map involving one freeway vs. another based on recent accident statistics rather than just distance and speeds.  

A similar philosophy could be used with respect to Bicycle Accidents.

**Comment by judopp** [comment]
I found the most wonderful database on traffic accidents:  https://cris.dot.state.tx.us/public/Query/#/public/welcome

My idea is to use this database to build a numerical model (regression) based on 5-10 features that predict whether an accident will occur (weather conditions, pavement, construction or not, day or night, age of driver, etc.).  Then we need to come up with an algorithm to put the different segments together.

**Comment by judopp** [comment]
I found the most wonderful database on traffic accidents: https://cris.dot.state.tx.us/public/Query/#/public/welcome

My idea is to use this database to build a numerical model (regression) based on 5-10 features that predict whether an accident will occur (weather conditions, pavement, construction or not, day or night, age of driver, etc.). Then we need to come up with an algorithm to put the different segments together.

**Comment by tanveerfc** [comment]
I was wondering if information related to the crime rates in any area could be added to the data-set/display. As a women driver, if driving in a new area I am always cautious about my safety. If information related to alternate routes is available I can make better choice. I do not want to divert the goal of the app very much, but just thought this could an added feature/information many could find useful.

**Comment by judopp** [comment]
Like this idea.

**Comment by meremaines** [comment]
I like that idea, too, Tanveer! I would think that would be fairly simple to do, too, and maybe could collaborate with others who went to the crime data jam earlier this year!

**Comment by tanveerfc** [comment]
I agree Meredith. It should be easy to implement as well. I would be curious to see options of crime data available to us on this.

**Comment by imaninotsalami** [comment]
It would be great if we also included options for biking/walking routes, especially if crime data can be included.

**Comment by judopp** [comment]
Also, weather conditions, night vs day. Many options can be offered/included.

**Comment by meremaines** [comment]
FYI, I also created a Slack channel within the Sketch City team if that's easier for us to all keep discussing. You can find it at #route-project. Loving all of these great ideas!

(If anyone needs a Slack invite, just enter your email address at http://sketchcity.herokuapp.com/ to automatically get added.)

**Comment by judopp** [comment]
All, a Team for this idea is forming and we welcome all interested members for ideas.  Look for me at the next few Tuesday evening meetings.

**Comment by judopp** [comment]
I have been analyzing the above Crime Data for many months now - and it turns out that streets in Midtown have the highest crime numbers.  At the same time, this part of the metro area ranks as one of the top 10 areas in Walkability according to comments made at a recent Sketch City Meeting on Mobility.  It would be interesting to study the issue further and sort out which is the cause and which is the effect.  In other words, is having people walking around (by themselves?) making them a crime victim or do the same features that make the area walkable make it accessible to perps?

**Comment by judopp** [comment]
At the same Sketch City Meeting on Mobility mentioned in the previous comment, there was an idea to include variability in the various modes of transportation used in the various internet "route-mapping" apps.  For example, instead of saying that this route will take 1 hour and 15 minutes - you could enter a confidence limit and the app could rank the various options that fit within this additional criterion.  Roads that have accidents frequently would be penalized relative to the "back way" which may be longer but more reliable.
